src/main/java/org/folio/edge/sip2/domain/messages/responses/ACSStatus.java

```diff
@@ -88,7 +88,7 @@ private ACSStatus(ACSStatusBuilder builder) {
     this.protocolVersion = builder.protocolVersion;
     this.institutionId = builder.institutionId;
     this.libraryName = builder.libraryName;
-    this.supportedMessages = Collections.unmodifiableSet(
+    this.supportedMessages =  Collections.unmodifiableSet(
```

Comment:
I think this is some sort of editor fail here. It seems that we are putting extra spaces when dealing with `supportedMessages`. We can probably revert these changes. I'm surprised checkstyle didn't flip out here.